pkg/services/pluginsintegration/pluginupdatechecker/service.go

```diff
@@ -82,3 +84,30 @@ func (s *Service) IsUpdatable(ctx context.Context, plugin pluginstore.Plugin) bo
 
 	return true
 }
+
+func (s *Service) CanUpdateMinor(currentVersion string, targetVersion string) bool {
```

Comment:
I'd name this `CanUpdate` and have a parameter, something like `onlyMinorVersions` to enable the behavior to ignore the major version or not.